base/pkg.jl

```diff
@@ -2,15 +2,19 @@
 
 module Pkg
 
-export Git, Dir, GitHub, Types, Reqs, Cache, Read, Query, Resolve, Write, Generate, Entry
+export Dir, GitHub, Types, Reqs, Cache, Read, Query, Resolve, Write, Generate, Entry
 export dir, init, rm, add, available, installed, status, clone, checkout,
        update, resolve, register, tag, publish, generate, test,
-       build, free, pin
+       build, free, pin, PkgError
 
-const DEFAULT_META = "git://github.com/JuliaLang/METADATA.jl"
+const DEFAULT_META = "https://github.com/JuliaLang/METADATA.jl"
```

Comment:
maybe trying to address https://github.com/JuliaLang/julia/issues/7005 simultaneously - should probably be done as a separate pr though